python-frontend/src/main/java/org/sonar/python/semantic/v2/converter/PythonTypeToDescriptorConverter.java

```diff
@@ -129,14 +129,20 @@ private static Descriptor convert(String moduleFqn, String parentFqn, String sym
       }
     }
 
+    var metaclassFQN = type.metaClasses()
+      .stream()
+      .map(metaClass -> typeFqn(moduleFqn, metaClass))
+      .findFirst()
+      .orElse(null);
+
     return new ClassDescriptor(symbolName, symbolFqn,
       superClasses,
       memberDescriptors,
       type.hasDecorators(),
       type.definitionLocation().orElse(null),
       hasSuperClassWithoutDescriptor,
       type.hasMetaClass(),
-      null,
+      metaclassFQN,
```

Comment:
Not strictly related to the PR, but I see the `metaclassFQN` method is not annotated as `CheckForNull` which is probably misleading.

Also, I'm a bit concerned that we're going to cement the usage of the `metaclassFQN` somewhat inconsistently (because `importType.importPath()` does not produce a fully qualified name according to our definition). I think it's okay for now, but we should keep in mind that this probably need to change eventually (like for decorators).